rest/service/identity.go

```diff
@@ -115,6 +115,7 @@ func CreateIdentity(userId string, skipCache bool) (models.UserIdentity, error)
 		FavoritePages:    []models.FavoritePage{},
 		SelfReport:       models.SelfReport{},
 		VisitedBundles:   nil,
+		ActiveWorkspace:  "foobar",
```

Comment:
@karelhala @fhlavac do you know what the default workspace ID should be for all users? I will update this PR accordingly. 

This ID will be used to seed existing users in our DB as well as any new users that are created. The workspace can then be changed later through the `update-active-workspace` rest endpoint.